src/jobservice/job/impl/default_context.go

```diff
@@ -49,7 +51,7 @@ func (dc *DefaultContext) Build(t job.Tracker) (job.Context, error) {
 	}
 
 	jContext := &DefaultContext{
-		sysContext: dc.sysContext,
+		sysContext: orm.NewContext(dc.sysContext, o.NewOrm()),
```

Comment:
Is transaction needed here? If it is, could you add a `TODO` here or open an issue?

Reply:
so far, no need transaction here, will add a TODO